scripts/dist.sh

```diff
@@ -2,10 +2,4 @@
 
 set -e
 
-export PATH=env/bin:${PATH}
-
-rm -rf dist/*
-
-pip install wheel
-
-python setup.py sdist bdist_wheel
+poetry publish --build
```

Comment:
Considering the command is now this simple, I think it's fine to just remove this script entirely. It was only there as a helper to _build_ the project, not fully publish it. But `poetry build` is just as simple!